.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+jobs:
+  check_links:
+    docker:
+      - image: circleci/node:lts
+    steps:
+      - run:
+          name: check links
+          command: npx broken-link-checker https://learn.madetech.com --exclude microsoft --exclude linkedin --exclude codewars.com -ro  
+workflows:
+  version: 2
+  # on_commit:
```

Comment:
I don't think the comments are necessary, let's leave it as weekly checks for now.